Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    postgres_to_redshift (0.5.0)
+    postgres_to_redshift (0.6.1)
```

Comment:
for some reason when I ran `bundle install` in the local version of the repo, it updated the gemfile.lock. Are these valid changes? I think so, right?